docs/Operators.md

```diff
@@ -17030,6 +17031,187 @@ expect(node, inputs=[x], outputs=[y],
 </details>
 
 
+### <a name="SoftmaxCrossEntropy"></a><a name="softmaxcrossentropy">**SoftmaxCrossEntropy**</a>
+
+  Loss function that measures the softmax cross entropy
+  between 'scores' and 'labels'.
+  The loss can be described as:
+      L = (l_1, l_2, ..., l_N), where N is the batch_size
+  
+  scores: (N, C) where C is the number of classes, or (N, C, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  labels: (N) where each value is 0 <= labels[i] <= C-1, or (N, d1, d2,..., dk),
+  	with K >= 1 in case of K-dimensional loss.
+  
+  The loss for one sample, l_n, can caculated as follows
+      let p = Softmax(scores)
+      l_n = -sum(label_i * log(p_i)), where i is the index of classes.
+  or
+      l_n = -sum(weight_i * label_i * log(p_i)), if 'weights' is provided.
+  Finally, L is reduced:
+  L = ReduceSum(L), if reduction = 'sum';
+      ReduceMean(L), if reduction = 'mean';
+      L, if reduction = 'none'
+
+#### Version
+
+This version of the operator has been available since version 12 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>reduction</tt> : string (default is mean)</dt>
+<dd>Type of reduction to apply to loss: none, sum, mean(default). 'none': no reduction will be applied, 'sum': the output will be summed. 'mean': the sum of the output will be divided by the number of elements in the output.</dd>
+</dl>
+
+#### Inputs (2 - 3)
+
+<dl>
+<dt><tt>scores</tt> : T</dt>
+<dd>The predicted outputs with shape [batch_size, class_size], or [batch_size, class_size, d1, d2 , ..., dk], where K is the number of dimensions.</dd>
+<dt><tt>labels</tt> : T</dt>
+<dd>The ground truth output tensor, same dimensions as 'scores'. Usualy, it's a one-hot representation of groud-truth class.</dd>
+<dt><tt>weights</tt> (optional) : T</dt>
```

Comment:
I feel it should be an attribute.

Reply:
in order to be consistent with MSELoss weight should be an input.